common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1519,13 +1563,43 @@ object CelebornConf extends Logging {
     conf.getTimeAsMs("rss.replicate.fastfail.duration", "60s")
   }
 
-  def maxPartitionNumSupported(conf: CelebornConf): Long = {
-    conf.getInt("rss.max.partition.number", 500000)
-  }
+  val SHUFFLE_FORCE_FALLBACK_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.shuffle.forceFallback.enabled")
+      .withAlternative("rss.force.fallback")
+      .categories("client")
+      .version("0.2.0")
+      .doc("Whether force fallback shuffle to Spark's default.")
+      .booleanConf
+      .createWithDefault(false)
 
-  def forceFallback(conf: CelebornConf): Boolean = {
-    conf.getBoolean("rss.force.fallback", false)
-  }
+  val SHUFFLE_FORCE_FALLBACK_PARTITION_THRESHOLD: ConfigEntry[Long] =
+    buildConf("celeborn.shuffle.forceFallback.numPartitionsThreshold")
+      .withAlternative("rss.max.partition.number")
+      .categories("client")
+      .version("0.2.0")
+      .doc(
+        "Celeborn will only accept shuffle of partition number lower than this configuration value.")
+      .longConf
+      .createWithDefault(500000)
+
+  val SHUFFLE_META_SERVICE_PORT: ConfigEntry[Int] =
+    buildConf("celeborn.shuffle.metaService.port")
+      .withAlternative("rss.driver.metaService.port")
+      .categories("client")
+      .version("0.2.0")
+      .doc("Port used by the LifecycleManager on the Driver.")
+      .intConf
+      .checkValue(
+        (port: Int) => {
+          if (port == 0) {
+            logWarning(
+              "The user specifies the port used by the LifecycleManager on the Driver, and its" +
+                s" values is $port, which may cause port conflicts and startup failure.")
+          }
```

Comment:
I guess the warning condition should be `port != 0` based on the message?

Reply:
Yea, updated